samples/openthread/coap_client/prj.conf

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# OpenThread settings and dependencies
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_POLL_MAX=4
+CONFIG_NET_CONNECTION_MANAGER=y
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_ENTROPY_GENERATOR=y
```

Comment:
Due to temporary issues we have with entropy subsystem and CC310, you might consider adding `CONFIG_ENTROPY_CC310=n` for now in the OpenThread samples.

Reply:
Added.